src/lib/template/templatizer.html

```diff
@@ -320,9 +320,13 @@
       // each row is a document fragment which is lost when we appendChild,
       // so we have to track each child individually
       var children = [];
+      var hide = host._hideTemplateChildren;
       for (var n = this.root.firstChild; n; n=n.nextSibling) {
```

Comment:
Consider giving TemplateInstance a `showHideChildren` method? Then use it here and in repeater's `_shotHideChildren`.